File: server/mc/common2/src/main/java/calebxzhou/rdi/mc/common2/tpa/TpaResult.java
package calebxzhou.rdi.mc.common2.tpa;

public record TpaResult(boolean success, String message) {
    public static TpaResult ok(String message) {
        return new TpaResult(true, message);
    }

    public static TpaResult error(String message) {
        return new TpaResult(false, message);
    }
}

File: mc/rcmd/src/main/kotlin/calebxzhou/rdi/mc/rcmd/tpa/TpaResult.kt
package calebxzhou.rdi.mc.rcmd.tpa


data class TpaResult(val success: Boolean, val message: String) {
    companion object {
        fun ok(message: String): TpaResult {
            return TpaResult(true, message)
        }

        fun error(message: String): TpaResult {
            return TpaResult(false, message)
        }
    }
}
